src/applications/disability-benefits/686c-674/analytics/helpers.js

```diff
@@ -0,0 +1,22 @@
+import recordEvent from 'platform/monitoring/record-event';
+
+export const fireAnalytics = submission => {
+  recordEvent({
+    event: 'disability-21-686c-submission-successful',
+    'disability-under18AndUnmarried':
+      submission['view:selectable686Options'].reportMarriageOfChildUnder18,
+    'disability-childAttendingSchool':
+      submission['view:selectable686Options']
+        .reportChild18OrOlderIsNotAttendingSchool,
+    'disability-reportingDivorce':
+      submission['view:selectable686Options'].reportDivorce,
+    'disability-stepchildLeftHousehold':
+      submission['view:selectable686Options'].reportStepchildNotInHousehold,
+    'disability-deathOfDependent':
+      submission['view:selectable686Options'].reportDeath,
+    'disability-marriageOfChild':
+      submission['view:selectable686Options'].reportStepchildNotInHousehold,
+    'disability-childStoppedAttendingSchool':
+      submission['view:selectable686Options'].reportStepchildNotInHousehold,
+  });
+};
```

Comment:
After our conversation on Friday about the payload getting mutated, looking at this makes me wonder: We know that `view` properties get stripped off the payload, if we fire this event on the confirmation page, we might have to change how this object is constructed before firing it off to analytics. I seem to remember when we were triaging the data duplication issue on Friday, selecting `reportDivorce` in the wizard eventually ended up being sent across as just `reportDivorce: true` instead of `data['view:selectable686Options'].reportDivorce`.

Reply:
So for the PR about the pair programming thing, I am accessing these directly on the form.data object so since they haven't been sent through the method to remove view fields it should not affect them